modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -6,20 +6,28 @@
 = Configuring OperatorHub for restricted networks
 
 Cluster administrators can configure OLM and OperatorHub to use local content in
-restricted network environments.
+a restricted network environment using a custom Operator catalog image. For this
```

Comment:
See previous comment. If `s/catalog/Catalog`, then apply throughout for consistency.